modules/common/src/main/scala/com/snowplowanalytics/snowplow/rdbloader/common/Common.scala

```diff
@@ -10,7 +10,7 @@
  * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
  * See the Apache License Version 2.0 for the specific language governing permissions and limitations there under.
  */
-package com.snowplowanalytics.snowplow.rdbloader.utils
+package com.snowplowanalytics.snowplow.rdbloader.common
```

Comment:
Needs to update the copyright to 2020 after this change

Reply:
Agreed.